plugins/basic/bootstrap/(zero-config)/bootstrap.ms

```diff
@@ -0,0 +1,39 @@
+-- Copyright (c) 2017 Shotgun Software Inc.
```

Comment:
would be great with a comment somewhere (not sure where the right place would be) to explain the meaning of the  `\(path)\` syntax.